website/pages/chapters/ridge.js

```diff
@@ -0,0 +1,845 @@
+            <em>square miles</em>. How would that change the intercept? How
+            would that change the slope?
+            <br /> <br />
+            What if instead, we changed the feature to <em>square inches</em>.
+          </TYU.Header>
+          <TYU.Explanation>
+            <p>
+              Regardless of if you are using square inches/feet/miles, 0 is
+              always 0 so the intercept should stay the same.
+            </p>
+            <p>
+              To think about how the scale of the features affects the slope
+              coefficient, it helps to think about two points and the line you
+              draw between them. So consider the houses{" "}
+              <IM math={`(1, 2050)`} /> and <IM math={`(100000, 50002000)`} />.
+              Remember for this conversation that the slope is the "rise over
+              run" between the points.
+            </p>
+
+            <h3>Square Miles</h3>
+            <p>
+              Since square miles are larger than square feet, when we convert
+              the units the magnitude of the features should decrease:{" "}
+              <IM math="100000" /> square feet is about <IM math="0.003" />{" "}
+              square miles. The dollar amounts are not changing with this
+              transformation, so the change in <IM math="y" /> (the "rise") is
+              the same. However, the change in <IM math="x" /> (the "run") is
+              much smaller since these numbers are tiny. With how fractions
+              work, decreasing the magnitude of the slope's denominator without
+              changing the numerator will cause the slope to increase (e.g.,
+              dividing by a smaller number).
+            </p>
+
+            <p>
+              Another way of thinking about this is to go back to the meaning of
+              the slope: "If I increase <IM math="x" /> by 1 unit, then the
+              price will change this much". If you increase the size of the
+              house by a square mile, you should expect that price to go up way
+              more than if you just increased it by one square foot. Therefore,
+              it makes sense that the coefficient if the unit was square miles
+              is higher.
+            </p>
+
+            <h3>Square Inches</h3>
+            <p>
+              Using the exact reasoning above, you should be able to convince
+              yourself that switching the scale to square inches will{" "}
+              <em>make the slope smaller</em>.
+            </p>
+          </TYU.Explanation>
+        </TYU>
+
+        <p>
+          When using multiple features (commonly called Multiple Linear
+          Regression), you can also usually interpret the coefficients in a
+          similar manner. Suppose for our housing example, we want to learn a
+          predictor of the form{" "}
+          <IM
+            math={`\\hat{y} = \\hat{w}_0 + \\hat{w}_1x[1] + \\hat{w}_2x[2]`}
```

Comment:
Ah, in our notation earlier we differentiate that `x_i` means the ith example from your dataset while `x[j]` is the jth data input for an example. I try as much as possible to avoid `x_{i,j}` since it's awfully confusing for newbies.

In fact, it's actually kind of wack that we use `w_0` instead of `w[0]`. Maybe something to fix in the future @_@